src/Microsoft.DotNet.Darc/src/DarcLib/AzureDevOpsClient.cs

```diff
@@ -647,9 +686,9 @@ public async Task<IList<Check>> GetPullRequestChecksAsync(string pullRequestUrl)
             string versionOverride = null,
             bool logFailure = true,
             string baseAddressSubpath = null,
-            int maxAttempts = 15)
+            int retryCount = 15)
```

Comment:
>retryCount [](start = 16, length = 10)

While I know retrying works (sometimes) I'm not a huge fan. I'd retry only certain cases, for example, if we are 400'd most likely it will fail the whole 15 times... Probably not a change for this PR, just wanted to note that

Reply:
This is something that @riarenas added recently.  I fixed it.  It actually couldn't have every worked because it was reusing the HttpRequestMessage.  @riarenas  do we need this at all?